Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1615,21 +1611,21 @@ def generate_message_names_implementation(receivers):
     result.append('\n')
     result.append('namespace IPC::Detail {\n')
     result.append('\n')
-    result.append('const MessageDescription messageDescriptions[static_cast<size_t>(MessageName::Count) + 1] = {\n')
+    result.append('const std::array messageDescriptions {\n')
```

Comment:
Hmm, this built on macOS but not glib ports unfortunately.

Reply:
C++ standardization lets us down; no big deal I guess since it’s generated code. Maybe giving the array type a name in the header then is a nice small cleanup (I suggested that in a separate comment)? So we don’t need to write it out here a second time. Not important before landing this change, obviously. Just a tiny possible tweak.

Reply:
>Maybe giving the array type a name in the header then is a nice small cleanup

This is what I did in the iteration that is currently up I believe.